### PR TITLE
Profile 2.0 navigation menu tweaks

### PR DESCRIPTION
**Pull request by erikphansen**
## Description
This addresses a few small requests from our designer.

## Testing done
Local

## Screenshots
image

## Acceptance criteria
- [x] mobile trigger button is left aligned 
- [x] menu item labels are updated
- [x] section headers align better vertically to top of sidenav menu

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/profile-2/components/AccountSecurity.jsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ class AccountSecurity extends Component {
       <>
         <h2
           tabIndex="-1"
-          className="vads-u-margin-y--2 medium-screen:vads-u-margin-y--4"
+          className="vads-u-line-height--1  vads-u-margin-y--2 medium-screen:vads-u-margin-y--4"
```
Comment (erikphansen) [inline src/applications/personalization/profile-2/components/AccountSecurity.jsx:24]: This is get the top of the section header to more closely align with the rule line at the top of the sidenav menu. Previously, with the line height of this header at its default value, the section header sat a lot lower than the sidenav menu rule line.

After:

Screen Shot 2020-05-04 at 10 36 24 AM

File: src/applications/personalization/profile-2/components/MobileMenuTrigger.jsx [hunk 2]
```diff
@@ -102,7 +102,7 @@ const MobileMenuTrigger = ({
   const buttonClasses = classnames({ fixed: isMenuTriggerPinned });
 
   return (
-    <div className="va-btn-sidebarnav-trigger">
+    <div className="va-btn-mobile-nav-trigger">
```
Comment (erikphansen) [inline src/applications/personalization/profile-2/components/MobileMenuTrigger.jsx:105]: Changing the class name for this React version of the sidenav. When we were using `va-btn-sidebarnav-trigger` some unwanted site-wide CSS was cascading down into this component.

File: src/applications/personalization/profile-2/components/MobileMenuTrigger.jsx [hunk 3]
```diff
@@ -114,10 +114,8 @@ const MobileMenuTrigger = ({
         onClick={openSideNav}
         ref={button}
       >
-        <span>
```
Comment (erikphansen) [inline src/applications/personalization/profile-2/components/MobileMenuTrigger.jsx:117]: this `span` isn't needed; it was copy/pasted from the site-wide side nav menu

File: src/applications/personalization/profile-2/sass/mobile-sidenav-trigger.scss [hunk 4]
```diff
@@ -7,6 +7,9 @@
     width: 100%;
     margin: 0;
     border-radius: 0;
+    display: flex;
```
Comment (erikphansen) [inline src/applications/personalization/profile-2/sass/mobile-sidenav-trigger.scss:10]: we were getting this behavior from sitewide styles applied to the `va-btn-sidebarnav-trigger` class

File: src/applications/personalization/profile-2/sass/mobile-sidenav-trigger.scss [hunk 5]
```diff
@@ -23,4 +26,7 @@
       width: auto;
     }
   }
+  @include media($medium-screen) {
```
Comment (erikphansen) [inline src/applications/personalization/profile-2/sass/mobile-sidenav-trigger.scss:29]: this was previously getting set by sitewide styles applied to the `va-btn-sidebarnav-trigger` class

File: src/applications/personalization/profile-2/components/AccountSecurity.jsx (hunk 1 above)
Comment (erikphansen) [reply src/applications/personalization/profile-2/components/AccountSecurity.jsx:24]: previously:

image

**Review by micahchiang — APPROVED** [review]
LGTM.

**Pull request merged**